webpack.config.js

```diff
@@ -52,7 +54,17 @@ export default (env, { mode }) => {
       splitChunks: {
         chunks: 'all'
       },
-      minimizer: ['...', new CssMinimizerPlugin()]
+      minimizer: [
+        '...',
+        new CssMinimizerPlugin({
+          minify: CssMinimizerPlugin.lightningCssMinify,
+          minimizerOptions: {
+            targets: browserslistToTargets(
+              browserslist('browserslist config, not maintained node versions')
+            )
```

Comment:
With `targets` not configured on the left, vs with the `targets` configured on the right

![image](https://user-images.githubusercontent.com/567105/190837937-1a55c51b-036e-4de3-b703-870ae8647f56.png)
![image](https://user-images.githubusercontent.com/567105/190837940-e3aba5ff-a901-4638-aa86-67ed4365ff7d.png)